cmd/payload-testing-prow-plugin/filetestresolver_test.go

```diff
@@ -55,11 +55,17 @@ func TestFileTestResolver(t *testing.T) {
 				}
 
 				actual, actualErr = testResolver.resolve("periodic-ci-openshift-api-master-build")
-				expected = api.MetadataWithTest{}
+				expected = api.MetadataWithTest{Metadata: api.Metadata{
+					Org:    "openshift",
+					Repo:   "api",
+					Branch: "master",
+				},
+					Test: "build",
+				}
 				if diff := cmp.Diff(expected, actual); diff != "" {
 					return fmt.Errorf("actual differs from expected: %s", diff)
 				}
-				if diff := cmp.Diff(fmt.Errorf("failed to resolve job periodic-ci-openshift-api-master-build"), actualErr, testhelper.EquateErrorMessage); diff != "" {
```

Comment:
Could you please add a new case for the changed code?
I would keep the coverage for the failure case.